html2asketch/nodeToSketchLayers.js

```diff
@@ -144,10 +152,26 @@ export default async function nodeToSketchLayers(node) {
       await style.addImageFill(backroundImageToUrl(backgroundImage));
     }
 
-    style.addBorder({color: borderColor, thickness: parseInt(borderWidth, 10)});
-
     if (boxShadow !== DEFAULT_VALUES.boxShadow) {
-      style.addShadow(shadowStringToObject(boxShadow));
+      const shadowObj = shadowStringToObject(boxShadow);
+
+      if (boxShadow.indexOf('inset') !== -1) {
+        if (borderWidth.indexOf(' ') === -1) {
+          shadowObj.spread += parseInt(borderWidth, 10);
+        }
+        style.addInnerShadow(shadowObj);
+      } else {
+        style.addShadow(shadowObj);
+      }
+    }
+
+    if (borderWidth.indexOf(' ') === -1) {
+      style.addBorder({color: borderColor, thickness: parseInt(borderWidth, 10)});
+    } else {
```

Comment:
Please add a comment here saying: `//support for one-side borders (using inner shadow because Sketch doesn't support that)`